src/MetricsUploader/proto/orbit_log_event.proto

```diff
@@ -20,6 +20,7 @@ message OrbitLogEvent {
     ORBIT_PRESET_LOAD = 6;
     ORBIT_PRESET_DELETE = 7;
     ORBIT_ITERATOR_ADD = 8;
+    ORBIT_ITERATOR_REMOVE = 9;
```

Comment:
Nit: As more metrics will be added in the future, may be we can put them in the alphabetically ordered without changing the numbering.

Reply:
I generally like the idea, I am going to do it the next time I add something and not in this PR. This is because there is a proto in g3, that is basically a copy of the one here. I will then change both protos, so they are in sync.